### ISSUE TITLE
bower package custom-selects not found

### ISSUE THREAD

**Issue by niklongstone** [Victoire/demo#10]
As per title the package doesn't exists.

I think `custom-select` could be the right alternative [similar packages](http://bower.io/search/?q=custom-s)

**Comment by paulandrieux** [comment]
Hi, thank you for your feedback,
@LoicGoyet I think you could take a look

**Comment by LoicGoyet** [comment]
@niklongstone Yeah sorry about that the repo just got migrated this week and unregistered from bower with the name `custom-selects` and re-registered with a new name : `trowel-select`. The issue is the same for `custom-radio` that is registered now as `trowel-radio`.

I'll fix the demo about those dependencies injection soon
